### ISSUE TITLE
[mechanics] [rogue]

### ISSUE THREAD

**Issue by zarkovod** [dalaranwow/dalaran-wow#5499]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1.  [Prey on the Weak] talent is currently not working every time on boss fights
it starts working only for Sinister Strike when the boss fight is reseted ( possible on Saurfang ) .
at the moment when it is working the average dmg of sinister strike is increased properly ( via crits  ).
it does not effect normal melee attacks .
spotted it when sudenly after reset fight on this boss my dps increased . and after the fight finished i have checked the average dmg on demage metter ( Skada ) it showed inceased sinisterstrike average dmg. but the melee attacks average was not changed.
 [Prey on the Weak] is stoping to work when during the fight have to change target ( adds spawn / ice tombs  and more ).
i'm not sure if the poisons crits should be effected from the talent (  [Prey on the Weak] ) too . poisons dmg is not getting changed too.
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by zarkovod** [comment]
i'm sorry for not posting meters in the note ( they got lost during the rest of the fights in icc )

**Comment by zarkovod** [comment]
[Prey on the Weak] does not effect eviscerate aither

**Comment by HiNico** [comment]
The assessment of [Prey on the Weak] here seems to reflect my experiance of abilities doing less than blizzlike damage.

**Comment by Jiaku** [comment]
It is not working properly, one attempt im doing 14k dps second attempt im doing 12k dps. (Even if i you assume i messed up with rotation, it can't 2k dps loss). And yes average crit of sinister strike is way lower when im doing 12k. When i'll have opportunity i'll make screenshots.
